src/main/java/com/google/sps/youtube/YoutubeResponse.java

```diff
@@ -0,0 +1,9 @@
+
+package com.google.sps.youtube;
+
+import java.io.IOException;
+import com.google.api.services.youtube.model.ChannelListResponse;
+
+public interface YoutubeResponse{
```

Comment:
all public classes and interfaces should have java docs

Reply:
Removed interface.